azurerm/config.go

```diff
@@ -33,6 +32,7 @@ import (
 	"github.com/Azure/azure-sdk-for-go/arm/resources/resources"
 	"github.com/Azure/azure-sdk-for-go/arm/resources/subscriptions"
 	keyVault "github.com/Azure/azure-sdk-for-go/dataplane/keyvault"
+	"github.com/Azure/azure-sdk-for-go/services/appinsights/mgmt/2015-05-01/insights"
```

Comment:
Given the package `insights` is used in multiple places within Azure (e.g. #478), I think it might be sensible to alias this import - I'll push a commit to update this